supplementary/alica_ros1/alica_ros_turtlesim/src/base.cpp

```diff
@@ -35,13 +35,6 @@ Base::Base(ros::NodeHandle& nh, ros::NodeHandle& priv_nh, const std::string& nam
     ac->setCommunicator<alicaRosProxy::AlicaRosCommunication>();
     ac->setTimerFactory<alicaRosTimer::AlicaRosTimerFactory>();
     ac->setLogger<alicaRosLogger::AlicaRosLogger>(agent_id);
-
-    // create world model
-    if (_loadDynamically) {
-        ALICASetWorldModel(nh, priv_nh);
-    } else {
-        ac->setWorldModel<turtlesim::ALICATurtleWorldModel>(nh, priv_nh);
```

Comment:
why does nh & priv_nh have to be passed in start? Can you not set the world model on the global blackboard here itself?